### ISSUE TITLE
Agency North/South Flag

### ISSUE THREAD

**Issue by secondhelpingsadmin** [megandalster/sh-homeplate#95]
To each donor/recipient new record, add a field, with box to right of Area: Label No/So; drop down values: North, South, blank.  Include in Search criteria; default to 'all'.
Use to identify agencies between north and south of the Broad River.

**Comment by secondhelpingsadmin** [comment]
Drop down menu: All, No, So.  Default is all.

**Comment by megandalster** [comment]
I would like to unify this with Delivery Area if possible.  that is, do we know which delivery areas are North of the Broad and which are South of the Broad.  Otherwise, this additional field seems a little redundant.  Advice?

**Comment by secondhelpingsadmin** [comment]
Delivery Area alone won't work... North or South have more than one Area, and some Areas may be split.  This is for the Board, none of whom know how to use HP, and some of whom only think in terms of North or South.  Need to dumb it down.

**Comment by megandalster** [comment]
Added No/So field to Client's profile, which can be set to North or South of the Broad.  Added the No/So dropdown to the Client Search and Reports pages, so that searches and reports on clients can be grouped this way.  Each client's No/So field will need to be edited by someone who know geography.  Default setting for this new field is blank.
